mongo/errors.go

```diff
@@ -252,9 +252,23 @@ type ServerError interface {
 	// HasErrorCodeWithMessage returns true if any of the contained errors have the specified code and message.
 	HasErrorCodeWithMessage(int, string) bool
 
+	// ErrorCodes returns a deduplicated list of error codes returned by the
```

Comment:
The latest revision does not deduplicate error codes.

```suggestion
	// ErrorCodes returns a list of error codes returned by the
```